packages/scoped-elements/README.md

```diff
@@ -0,0 +1,261 @@
+Then, the `html` function provided by the `createScopedHtml` method transforms the template literal into another one replacing the tags used by the user by the ones defined by the custom elements. Finally, the transformed template literal is going to be processed by `lit-html`.
+
+`<my-button>${this.text}</my-button>`
+
+becomes:
+`<my-button-3>${this.text}</my-button-3>`
+
+## Usage
+
+1. Import `createScopedHtml` from `@open-wc/scoped-elements`.
+
+   ```js
+   import { createScopedHtml } from '@open-wc/scoped-elements';
+   ```
+
+2. Import the classes of the components you want to use.
+
+   ```js
+   import MyButton from './MyButton.js';
+   import MyPanel from './MyPanel.js';
+   ```
+
+3. Create the `html` function and define the tags you want to use for your
+   components.
+
+   ```js
+   const html = createScopedHtml({
+     'my-button': MyButton,
+     'my-panel': MyPanel,
+   });
+   ```
+
+4. Use your components in your html.
+
+   ```js
+   render() {
+     return html`
+       <my-panel class="panel">
+         <my-button>${this.text}</my-button>
+       </my-panel}>
+     `;
+   }
+   ```
+
+### Complete example
+
+```js
+import { css, LitElement } from 'lit-element';
+import { createScopedHtml } from '@open-wc/scoped-elements';
+import MyButton from './MyButton.js';
+import MyPanel from './MyPanel.js';
+
+const html = createScopedHtml({
+  'my-button': MyButton, // <-- binds the element to a tag in your html
+  'my-panel': MyPanel,
+});
+
+export default class MyElement extends LitElement {
+  static get styles() {
+    return css`
+      .panel {
+        padding: 10px;
+        background-color: grey;
+      }
+    `;
+  }
+
+  static get properties() {
+    return {
+      text: String,
+    };
+  }
+
+  render() {
+    return html`
+      <my-panel class="panel">
+        <my-button>${this.text}</my-button>
+      </my-panel}>
+    `;
+  }
+}
+```
+
+## Limitations
+
+1. Components imported via npm **MUST NOT** be self registering components. If a shared component (installed from npm) does not offer an export to the class alone, without the registration side effect, then this component can not be used. E.g. every component that calls `customElement.define`
+
+   ```js
+   export class MyEl { ... }
+   customElement.define('my-el', MyEl);
+   ```
+
+   Or uses the `customElement` typescript decorator
+
+   ```ts
+   @customElement('my-el')
+   export class MyEl { ... }
+   ```
+
+   Only side effects free class exports may be used
+
+   ```js
+   export class MyEl { ... }
+   ```
+
+   Note: not share or components within the main app shell are exempted from this rule.
+
+2. Every (!!!) component that uses sub components need to use `scoped-elements`. Any import to a self registering component can result in a browser exception - completely breaking the whole application.
+
+3. Currently, only `lit-html` is supported (though other rendering engines could be incorporated in the future)
+
+4. You can not use tag selectors in css
```

Comment:
```suggestion
4. You can not use tag selectors in css, but you could use a class name instead
```

Reply:
I changed it to

4. You can not use tag selectors in css, but you could use an id, a class name or even a property instead.